src/OrchardCore.Modules/OrchardCore.Media.Azure/Startup.cs

```diff
@@ -56,5 +105,15 @@ public override void ConfigureServices(IServiceCollection services)
                 options.Filters.Add(typeof(MediaBlobStorageOptionsCheckFilter));
             });
```

Comment:
While we're discussing this, I wonder about this filter.

It has to run on every request, and all it does is provide a notification in the admin if the `connectionString` is empty. We could still do the `connectionString` check once on Startup, or enable the feature, show the notification that it's not configured, without having to have the filter?

Reply:
Yep, just do the check on startup, and log something as Error.

Reply:
Done